app/src/main/java/ch/epfl/sdp/appart/CameraActivity.java

```diff
@@ -29,6 +29,9 @@
 import androidx.core.content.ContextCompat;
 
 import ch.epfl.sdp.appart.database.DatabaseService;
+import ch.epfl.sdp.appart.glide.visitor.GlideImageViewLoader;
+import ch.epfl.sdp.appart.user.UserViewModel;
+import com.bumptech.glide.Glide;
```

Comment:
I think these are not used imports but maybe I am wrong.

Reply:
yes I've removed all unnecessary import!